### PR TITLE
prunes the JSON tree

### PR DESCRIPTION
**Pull request by logicerpsolution**
#564

I have putted my code in base/utils.js.

### CODE REVIEW

**Comment by arthurcamara1** [comment]
What problem is this solving and where is it going to be used? Just trying to clarify the context for a `prune` function. Is there an issue number related to it?

And take a look at https://github.com/logicerpsolution/vizabi/blob/develop/src/base/utils.js#L939
```javascript
var allowed_IDs = ["time", "gdp_per_cap", "gdppc_pday"];
```
The columns should be configurable, not hardcoded, so they might be passed as an argument to the function you want to implement.

And not sure why, but the buid failed. Does `gulp build` work locally for you?

File: src/base/utils.js [hunk 1]
```diff
@@ -920,6 +920,45 @@
...
+		var result={"id":indicatorsTree.id,"children":[]};
+		for(var i=0; i<indicatorsTree.children.length;i++)
+		 {
+		  if(indicatorsTree.children[i] !=undefined)
+			findNode.call(indicatorsTree.children[i],indicatorsTree.children[i].id,result);
+			return result;
+		 }
+	},
+	function findNode(id,result) {
+	  	var allowed_IDs = ["time", "gdp_per_cap", "gdppc_pday"];
```
Comment (valorkin) [inline src/base/utils.js]: @arthurcamara1 hardcoded indicator names?

**Comment by angieskazka** [comment]
@arthurcamara1 @valorkin please don't look here until i approved this. 
the related issue is #564

**Comment by arthurcamara1** [comment]
Cool @angieskazka

**Pull request closed without merging**